clusterloader2/cmd/clusterloader.go

```diff
@@ -31,38 +33,70 @@ import (
 )
 
 var (
-	kubeConfigPath = flag.String("kubeconfig", "", "path to the kubeconfig file")
-	testConfigPath = flag.String("testconfig", "", "path to the test config file")
+	clusterConfig  api.ClusterConfig
+	testConfigPath string
 )
 
-func validateFlags() []error {
+func initClusterFlags() {
+	flag.StringVar(&clusterConfig.KubeConfigPath, "kubeconfig", "", "path to the clusterconfig file")
+	flag.StringVar(&clusterConfig.Provider, "provider", "", "kubernetes provider")
+	flag.StringVar(&clusterConfig.Host, "host", "", "host ip address")
```

Comment:
You can copy over these flags and their descriptions from test/e2e/framework/test_context.go.

Reply:
ok